src/rpc/request.rs

```diff
@@ -525,23 +525,35 @@ pub enum Request {
     String(String),
 
     #[api(type = 1002)]
-    #[display("progress({0})")]
+    #[display("progress: {0}")]
     Progress(Progress),
 
     #[api(type = 1005)]
-    #[display("swap_progress({0})", alt = "{0:#}")]
+    #[display(inner)]
     SwapProgress(SwapProgress),
 
     #[api(type = 1003)]
     #[display("read_progress({0})")]
     ReadProgress(SwapId),
 
+    #[api(type = 9999)]
+    #[display("test_progress({0})")]
+    TestProgress(SwapId),
```

Comment:
Unused

Reply:
Fixed in 109f6125744e7ac9a651cc08cb7f78d82606c37c